Content.Shared/Doors/Components/FirelockComponent.cs

```diff
@@ -58,21 +51,22 @@ public sealed partial class FirelockComponent : Component
         public TimeSpan? EmergencyCloseCooldown;
 
         /// <summary>
-        /// Whether the firelock can open, or is locked due to its environment.
+        /// Whether or not a pressure difference exceeding the pressure threshold exists around the firelock.
         /// </summary>
-        public bool IsLocked => Pressure || Temperature;
+        [DataField, AutoNetworkedField]
+        public bool Pressure;
 
         /// <summary>
-        /// Whether the firelock is holding back a hazardous pressure.
+        /// Whether or not the airlock is commanded by an air alarm to close.
         /// </summary>
         [DataField, AutoNetworkedField]
-        public bool Pressure;
+        public bool ExtLocked;
 
         /// <summary>
-        /// Whether the firelock is holding back extreme temperatures.
+        /// Whether the firelock can open, or is locked due to its environment. Note that even when locked,
+        /// the firelock can still be pried, so this should be more accurately named "WantsToClose".
         /// </summary>
-        [DataField, AutoNetworkedField]
-        public bool Temperature;
+        public bool IsLocked => ExtLocked || Pressure;
```

Comment:
I want to avoid changing public fields that may have associated data fields for purely cosmetic reasons. Changing variable names also clutters the diff.